public/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```

Comment:
Should we include the name of the site here, or is this primarily for icons?

Reply:
fwiw it looks like whatever is set as the short_name is used when saving to an iPhone

Reply:
discussed in eng sync 🎉